src/commands/view/ShowOffset.ts

```diff
@@ -18,6 +18,10 @@ export default {
     const zoom = this.em.getZoomDecimal();
     const el = opt.el as HTMLElement | undefined;
 
+    if (!(el instanceof HTMLElement)) {
+      return;
+    }
```

Comment:
Any valid reason for this check? 
I don't think there are other possible "visible" nodes beside the textnode, so the checks below should be enough already.

Reply:
This is to avoid further processing any non-HTMLElement nodes. No reason to waste the cycles, and it could prevent future errors further down the pipe if there are checks missed. 

Side note, the type checking of the comment node later in the code to an HTMLElement appears to be syntactical sugar gone wrong. As in the JS transpiled code no actual type checking was happening, and hence the error. Note sure were the transpile error was happening, but the parser found it. Leave it to me to find the bugs in the tools... smh